cvat/requirements/base.txt

```diff
@@ -1,11 +1,13 @@
-# SHA1:72d30d3a78182797ba879af4b7b50f17ecaf2e8f
+# SHA1:e320604118ebbbb6f9a0460d977e416d4cb27200
 #
 # This file is autogenerated by pip-compile-multi
 # To update, run:
 #
 #    pip-compile-multi
 #
 -r ../../utils/dataset_manifest/requirements.txt
+--no-binary lxml
+--no-binary xmlsec
```

Comment:
Please remove these changes in requirements, except `production`. This is handled manually for now, it's a compatibility problem in the requirements update command we use currently.

Reply:
Removed